radmsg.c

```diff
@@ -285,7 +285,7 @@ struct radmsg *buf2radmsg(uint8_t *buf, int len, uint8_t *secret, int secret_len
     if (secret && buf[0] == RAD_Accounting_Request) {
         memset(auth, 0, 16);
         if (!_validauth(buf, len, auth, secret, secret_len)) {
-            debug(DBG_WARN, "buf2radmsg: Accounting-Request message authentication failed");
+            debug(DBG_WARN, "buf2radmsg: Accounting-Request message-authenticator invalid");
```

Comment:
this may be a bit confusing: in this section we check the authenticator in the RADIUS header (a special case of the check on line 293). In the RFC the field is just called 'Authenticator', depending on direction, its sometimes refered to as the 'Request Authenticator' or 'Response Authenticator'.

No need to update the PR, I will try to restructure this part so it becomes clearer.